### ISSUE TITLE
attribute selector error

### ISSUE THREAD

**Issue by Anonyfox** [Gonzih/crabquery#2]
When selecting `"meta[property=\"og:title\"]"` from the following document 

```html
<head>
    <title>... snip ...</title>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <!-- ... snip ... -->
    <meta property="og:title" content="... snip ..." />
<!-- ... snip ... -->
```

I get all metatags as the result vector, not just the one actually matching "og:title" as I would expect.

**Comment by Gonzih** [comment]
Fixed and released new version 0.1.7.

Thanks a lot for reporting this!

**Comment by Anonyfox** [comment]
Thank you for the fast fix!
